kb/how-to-troubleshoot-last-connection-error.md

```diff
@@ -0,0 +1,75 @@
+---
+slug: troubleshoot-last-connection-error
+title: Troubleshoot the "failed reaching server" error
+tags:
+  - cloud
+  - namespaces
+date: 2023-05-09T00:00:00Z
+---
+
+The error, `failed reaching server: last connection error`, is usually caused by an expired TLS certification.
```

Comment:
Could consider flipping the statement:

An expired TLS certification can often cause `failed reaching server: last connection error`